keywords/LiteServiOS.py

```diff
@@ -70,6 +154,44 @@ def remove(self):
         if bundle_id in output:
             raise LiteServError("LiteServ-iOS is still present after uninstall")
 
+    def remove(self):
+        """
+        Remove the iOS app from the simulator
+        """
+        bundle_id = "com.couchbase.LiteServ-iOS"
+        if self.storage_engine == "SQLCipher":
+            bundle_id = "com.couchbase.LiteServ-iOS-SQLCipher"
+
+        log_info("Removing LiteServ")
+
+        self.stop()
+
+        # Get the device ID
+        device_id = None
+        list_output = subprocess.Popen(["xcrun", "simctl", "list"], stdout=subprocess.PIPE)
```

Comment:
I was not able to successfully launch multiple simulators, though some articles on the internet mentioned it is possible.